src/components/_fs-aside.scss

```diff
@@ -6,15 +6,18 @@
  */
 
 .fs-aside {
-  width: 22.85714%;
```

Comment:
Not sure how these specific widths were calculated before but we should have grid-template-columns in `.fs-container` match them by default to keep the layout the same by default.

Reply:
22.85714% 74.28571%